starwars_api/models.py

```diff
@@ -11,15 +11,20 @@ def __init__(self, json_data):
         Dynamically assign all attributes in `json_data` as instance
         attributes of the Model.
         """
-        pass
+        self.json_data = json_data
```

Comment:
I don't think you ever use the attribute self.json_data so probably not necessary to save it as a class attribute